src/apps/vpn/telemetry.cpp

```diff
@@ -148,3 +160,22 @@ void Telemetry::periodicStateRecorder() {
   }
 }
 #endif
+
+#if defined(MZ_ANDROID)
+void Telemetry::onDaemonStatus(const QString& data) {
+  auto doc = QJsonDocument::fromJson(data.toUtf8());
+  bool connected = doc.object()["connected"].toBool(false);
+  if (!connected) {
+    // If we're not connected, connection health is irrelevant.
+    return;
+  }
+  auto status = doc.object()["connection-health-status"].toString();
+  if (status.isNull()) {
+    return;
+  }
+
+  mozilla::glean::sample::android_daemon_status.record(
```

Comment:
Just to be on the safe side, let's also add the Glean.js collection since this is going out in 2.13 still.

Reply:
is glean-js now active again on android ?

Reply:
Argh, I _think_ I did enable it and I just realized that was DUMB. Nevermind me, PR incoming.

Reply:
Rollercoaster of emotions, I thought I had removed the Android conditions BUT I DID NOT. You are correct, no Glean.js on Android. Phew!